src/PageHeader/PageHeader.examples.stories.tsx

```diff
@@ -67,36 +69,35 @@ WebhooksOnNarrowViewport.parameters = setViewportParamToNarrow
 export const PullRequestPage = () => (
   <Box sx={{padding: 3}}>
     <PageHeader>
-      <PageHeader.ContextArea>
-        <PageHeader.ParentLink href="http://github.com">Pull requests</PageHeader.ParentLink>
-      </PageHeader.ContextArea>
       <PageHeader.TitleArea>
         <PageHeader.Title as="h1">
           PageHeader component initial layout explorations extra long pull request title
         </PageHeader.Title>
-        <PageHeader.Actions>
-          <Hidden when={['regular', 'wide']}>
-            <ActionMenu>
-              <ActionMenu.Anchor>
-                <IconButton aria-label="More pull request actions" icon={KebabHorizontalIcon} />
-              </ActionMenu.Anchor>
-              <ActionMenu.Overlay width="small">
-                <ActionList>
-                  <ActionList.Item onSelect={() => alert('Edit button action')}>Edit</ActionList.Item>
-                  <ActionList.Item onSelect={() => alert('Code button action')}>Code</ActionList.Item>
-                </ActionList>
-              </ActionMenu.Overlay>
-            </ActionMenu>
-          </Hidden>
-
-          <Hidden when={['narrow']}>
-            <Box sx={{display: 'flex'}}>
-              <Button>Edit</Button>
-              <Button leadingVisual={CodeIcon}>Code</Button>
-            </Box>
-          </Hidden>
-        </PageHeader.Actions>
       </PageHeader.TitleArea>
+      <PageHeader.ContextArea>
+        <PageHeader.ParentLink href="http://github.com">Pull requests</PageHeader.ParentLink>
+      </PageHeader.ContextArea>
+      <PageHeader.Actions>
+        <Hidden when={['regular', 'wide']}>
+          <ActionMenu>
+            <ActionMenu.Anchor>
+              <IconButton aria-label="More pull request actions" icon={KebabHorizontalIcon} />
+            </ActionMenu.Anchor>
+            <ActionMenu.Overlay width="small">
+              <ActionList>
+                <ActionList.Item onSelect={() => alert('Edit button action')}>Edit</ActionList.Item>
+                <ActionList.Item onSelect={() => alert('Code button action')}>Code</ActionList.Item>
+              </ActionList>
+            </ActionMenu.Overlay>
+          </ActionMenu>
+        </Hidden>
+        <Hidden when={['narrow']}>
+          <Box sx={{display: 'flex'}}>
```

Comment:
We need a gap here to add space between the two buttons

<img width="1791" alt="Screenshot 2023-12-13 at 6 13 38 PM" src="https://github.com/primer/react/assets/2313998/0bb86c58-c6e0-48d5-995d-a7ac2703f5d0">

Reply:
done!